docs/testing.rst

```diff
@@ -68,6 +69,37 @@ The :ref:`Project` object for your project.
         assert 'MyContract' in project.compiled_contract_data
 
 
+How populus finds the project of the project fixture
+''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+If no other argument is provided, populus assumes that the tested project directory is the one where the tests run from.
+This is true if you run py.test from within the project's directory, or with a positional argument to this projecr's directory,
+e.g. ``$ py.test /path/to/my/project/``.
```

Comment:
I'd change this to `$ py.test tests/` so that the positional argument is a relative path.

Reply:
different case actually